group_vars/orangelight/alma_prod.yml

```diff
@@ -16,13 +16,14 @@ passenger_ruby: "/usr/bin/ruby2.6"
 ruby_version_override: "ruby2.6"
 bundler_version: "2.1.4"
 postgresql_is_local: false
-postgres_host: "lib-postgres-prod1"
-postgres_version: 13
-postgres_admin_user: "postgres"
-ol_db_host: "{{ postgres_host }}"
-ol_db_name: "{{ vault_ol_alma_qa_db_name }}"
-ol_db_user: "{{ vault_ol_alma_qa_db_user}}"
-ol_db_password: "{{ vault_ol_alma_qa_db_password }}"
+postgres_host: lib-postgres3.princeton.edu
+postgres_admin_user: 'postgresadmin'
+postgres_admin_password: '{{ vault_postgres_admin_password }}'
+postgres_version: 10
```

Comment:
This doesn't seem like the right version?

Reply:
```
pulsys@lib-postgres3:~$ postgres -V
postgres (PostgreSQL) 10.17 (Ubuntu 10.17-1.pgdg16.04+1)
```

Reply:
That's for the client? @kayiwa is this right?

Reply:
That's the server - psql would show the client version.

Reply:
Also, `SELECT version();` run via psql on Orangelight production does the following:

```
PostgreSQL 10.17 (Ubuntu 10.17-1.pgdg16.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609, 64-bit
```

Reply:
Okay!! I think I got confused because it was changing from "lib-postgres-prod1" to "lib-postgres3" -- but this is reverting back to using the old database so right, yes.